pkg/uprobetracer/ldcache_parser.go

```diff
@@ -0,0 +1,199 @@
+package uprobetracer
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"strings"
+	"unsafe"
+)
+
+const (
+	cache1Header    string = "ld.so-1.7.0"
+	cache1HeaderLen int    = 11
+
+	cache2Header    string = "glibc-ld.so.cache"
+	cache2HeaderLen int    = 17
+	cache2Version   string = "1.1"
+
+	ldCache1EntrySize uint32 = 12
+	ldCache1Size      uint32 = 16
+	ldCache2EntrySize uint32 = 24
+	ldCache2Size      uint32 = 48
+)
+
+type ldCache1Entry struct {
+	Flags int32
+	Key   uint32
+	Value uint32
+}
```

Comment:
Is there a documentation about this struct and the following ones you can mention in godoc comments?

Reply:
Sadly, I didn't find any doc. I searched in Google, and the glibc document, but there's no doc about this. The only doc is the glibc source code.

Reply:
The godoc comments is above the `parseLdCache` entry function.